### PR TITLE
Minor Linting fixups - 2022-08-05

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

Minor linting fixups

- docs linting and cleanup
- unused variables
- unused import
- "mutually_exclusive" which was accidentally dropped on the floor.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

plugins/modules/cloudformation.py
plugins/modules/ec2_eni.py
plugins/modules/ec2_eni_info.py
plugins/modules/ec2_instance.py
plugins/modules/ec2_security_group.py
plugins/modules/ec2_vol.py
plugins/modules/ec2_vol_info.py
plugins/modules/ec2_vpc_dhcp_option.py
plugins/modules/ec2_vpc_net.py
plugins/modules/ec2_vpc_subnet.py
plugins/modules/elb_classic_lb.py
plugins/modules/s3_bucket.py
plugins/modules/s3_object.py

##### ADDITIONAL INFORMATION

Removes some documentation references to old "versions", these were Ansible versions which are long out of support.

CloudFormations module previously talked about keeping boto3 and botocore up to date.  Now that we have an official policy around supported versions this is no longer as relevant and the module itself is no longer changing as fast.

### CODE REVIEW

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Review by goneri — APPROVED** [review]

File: changelogs/fragments/954-linting.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+breaking_changes:
```
Comment (goneri) [inline changelogs/fragments/954-linting.yml:1]: It may be nice to isolate this specific change in a different PR.

Comment (tremble) [reply changelogs/fragments/954-linting.yml:1]: I'll strip it from the backports

**Pull request merged**